clang/include/clang/CIR/Interfaces/CIRLoopOpInterface.td

```diff
@@ -96,4 +96,11 @@ def LoopOpInterface : OpInterface<"LoopOpInterface", [
   }];
 }
 
+def ConditionOpInterface : OpInterface<"ConditionOpInterface", []> {
```

Comment:
We should probably name this something more CIR specific: `CIR_ConditionOpInterface`. @xlauko any naming suggestions since you've been renaming a bunch?

Reply:
Yes everything in tablegen needs to be prefixed with the CIR_ so it does not create name clashes for potential users or if core decides to add `ConditionOpInterface`. Unfortunately tablegen does not have namespace so this is the way.